lib/Runtime/Language/JavascriptConversion.h

```diff
@@ -12,10 +12,13 @@ namespace Js {
         static Var OrdinaryToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * scriptContext);
         static Var MethodCallToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * scriptContext);
         static Var ToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * scriptContext);
-        static BOOL CanonicalNumericIndexString(Var aValue, double *indexValue, ScriptContext * scriptContext);
+        static BOOL CanonicalNumericIndexString(JavascriptString *aValue, double *indexValue, ScriptContext * scriptContext);
 
         static void ToPropertyKey(Var argument, ScriptContext* scriptContext, const PropertyRecord** propertyRecord);
 
+        static PropertyQueryFlags BooleanToPropertyQueryFlags(BOOL val) { return val ? Property_Found : Property_NotFound; }
+        static BOOL PropertyQueryFlagsToBoolean(PropertyQueryFlags val) { return val == Property_Found; }
```

Comment:
>BOOL PropertyQueryFlagsToBoolean [](start = 15, length = 32)

minor naming nit: (semantically this is fine) -- but instead of ToBoolean perhaps this should be ToBOOL for the same reasons that we make a distinction between BOOL and bool for various API return types. It seems that the types involved here are all BOOL and it might be good to capture that.

Although, IMO this name looks and reads better.